lib/common/formatHelpers/setSwiftFileProperties.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+/**
+ * Outputs an object with swift format configurations. Sets import, object type and access control.
+ * @memberof module:formatHelpers
+ * @param {Object} file - The file object declare at configuration
+ * @param {String} objectType - Type of the swift object.
+ * @param {String} transformGroup - The transformGroup of the file, so it can be applied proper import
+ * @returns {Object}
+ */
+function setSwiftFileProperties(file, objectType, transformGroup) {
```

Comment:
We have been trying to move to using the `options` object on the file to hold configuration like this. If you see in other formats and format helpers, things like `showFileHeader` and `outputReferences` are in an `options` object. Could we do something like that here to be consistent?

Reply:
i updated that (: what do you think?